compiler/prelude/prelude_min.go

```diff
@@ -0,0 +1,27 @@
+// Code generated by genmin. DO NOT EDIT.
```

Comment:
IIUC, the last period is not needed:

https://github.com/golang/go/issues/13560#issuecomment-276866852

Reply:
Fixed.